charts/redpanda/templates/statefulset.yaml

```diff
@@ -64,7 +64,7 @@ spec:
               rpk redpanda tune all
           securityContext:
             capabilities:
-              add: ["CAP_SYS_RESOURCE"]
```

Comment:
Can you add a comment here along with the link you provided. This would help cover future questions about this. Thank you.